### DISCUSSION TITLE
API key QR codes using local IP address

### DISCUSSION THREAD

**Discussion by ElGranBellini** [alfio-event/alf.io#1234] [Q&A]
Hello!
I'm running Alfio on a local server. 
When generating an API key (to use Alfio mobile app) the QR code generated embeds the server's local IP address in baseUrl instead of the FQDN. Then, of course, the mobile app can't register the account. 

I have defined the Base application url in system configuration.

Ex QR:
{"apiKey":"cf.....",**"baseUrl":"192.168.x.x"**}

Any ideas?
Thank you!

**Comment by aknocs** [comment]
Hi.

I'm running into the same problem. Any advancement?

Thanks in advance.

**Comment by cbellone** [comment]
Hi,
at the moment the QR Code is generated in the browser, and the baseUrl reflects the current admin address.
So if you want your QR Code to contain the public address, just access the admin from that address.
If for any reasons you can't do that, you can download the QR Code using the "Download all API Keys" button.

**Reply by aknocs** [reply]
Found this [googlegroups](https://groups.google.com/g/alfio/c/hlvLAm915HY) comment about the same issue and managed to make it work. Thank you for answering so quickly.

**Reply by cbellone** [reply]
NP. Please consider marking my comment as "answer", so that it could help other people find the solution sooner.
Thanks
